pkg/sql/backfill.go

```diff
@@ -1069,9 +1070,17 @@ func (sc *SchemaChanger) validateInvertedIndexes(
 	countReady := make([]chan struct{}, len(indexes))
 
 	for i, idx := range indexes {
+		// Shadow i and idx to prevent the values from changing within each
+		// gorountine.
 		i, idx := i, idx
 		countReady[i] = make(chan struct{})
 
+		// Skip partial indexes for now.
+		// TODO(mgartner): Validate partial inverted index entry counts.
+		if idx.IsPartial() {
```

Comment:
can you have a partial inverted index?

Reply:
At the moment they aren't fully supported. They likely won't make it into 20.2, and so I'll probably end up disallowing their creation for the release.